### PR TITLE
feat: email summary to use mailx

### PR DESCRIPTION
**Pull request by kcho**
Pronet has added `mailx` to the dev and production server. Switching to `mailx` email summary by default instead of gmail SMTP.

### CODE REVIEW

**Comment by tashrifbillah** [comment]
Never mind, I see that you have retained the Google option.

File: lochness/email/__init__.py [hunk 1]
```diff
@@ -104,7 +108,8 @@ def send_detail_google(Lochness,
     s.quit()
 
 
-def send_out_daily_updates(Lochness, days: int = 1, test: bool = False):
+def send_out_daily_updates(Lochness, days: int = 1,
+                           test: bool = False, mailx: bool = True):
```
Comment (tashrifbillah) [inline lochness/email/__init__.py:112]: Okay I understand the use of `mailx` bool but where did we use this new function? I see one use in `test_email.py` but none in the main code.

Comment (kcho) [reply lochness/email/__init__.py:112]: it's under `scripts/sync.py`

Comment (tashrifbillah) [reply lochness/email/__init__.py:115]: Okay then why is there no update in `scripts/sync.py` in this PR?

**Pull request merged**